lib/active_encode/engine_adapters/media_convert_adapter.rb

```diff
@@ -0,0 +1,358 @@
+# frozen_string_literal: true
+require 'active_encode/engine_adapters/media_convert_output.rb'
+require 'active_support/core_ext/integer/time'
+require 'addressable/uri'
+require 'aws-sdk-cloudwatchevents'
+require 'aws-sdk-cloudwatchlogs'
+require 'aws-sdk-mediaconvert'
+require 'file_locator'
+
+require 'active_support/json'
+require 'active_support/time'
+ActiveSupport.parse_json_times = true
+Time.zone = 'America/Chicago'
```

Comment:
I'm concerned about setting these globally for all users on class load.  Is there a way these can be set on the just the adapter or instructions to set these in an initializer where the ActiveEncode adapter is configured?

Reply:
Good question. I'll add it to the instructions. The time zone should probably match what the consumer is using elsewhere, though it probably doesn't matter a whole lot. Within the context of a Rails application, it should already be set anyway. I'll look into whether `parse_json_times` can be passed as an option at decode time instead of being set globally.

Reply:
Turns out these weren't needed at all. They were left over from when I was serializing live responses to use as test fixtures.